src/chainlifter/lifter.py

```diff
@@ -100,4 +100,10 @@ def convert_coordinate(
         :param strand: query strand (``"+"`` by default).
         :return: first match TODO return whole list
```

Comment:
TODO

Reply:
good call. gonna take a look in https://github.com/GenomicMedLab/chainlifter/issues/12